xarray/core/dataset.py

```diff
@@ -1722,12 +1722,22 @@ def to_zarr(
         consolidated: bool = False,
         append_dim: Hashable = None,
         region: Mapping[str, slice] = None,
+        safe_chunks: bool = True,
     ) -> "ZarrStore":
         """Write dataset contents to a zarr group.
 
-        .. note:: Experimental
-                  The Zarr backend is new and experimental. Please report any
-                  unexpected behavior via github issues.
```

Comment:
is this something we should do in a separate PR, or should we remove all similar warnings here, too? Also, should we add a entry to `whats-new.rst` for this?

Reply:
I have updated whats-new.

I agree that we should remove the notes about Zarr be experimental.